index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Baloo+Da+2:wght@400;500;600;700;800&family=Josefin+Slab:ital,wght@0,300;0,400;0,500;1,300;1,400&family=Mulish:ital,wght@0,300;0,400;0,500;0,600;1,200;1,300;1,400;1,500&family=Poppins:ital,wght@0,200;0,300;1,100;1,200;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <div class="container">
+<header>
+        <a href="#" class="logo">My Logo</a>
+        <nav>
+         <img class="onion"  src="/portfolioimgs/Union.svg" alt="navButton">
+        </nav>
+        </header>
+    <section>
+       <h2  class="headline1">I'm Loukmane
+        Glad to see you!</h2>
+        <p class="text">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="contact">LET'S CONNECT</p><br><img class="socialimgs" src="/portfolioimgs/socialMedia.svg" alt="contact">
+       </section>
+
+</div>
+</body>
+</html>
```

Comment:
- [ ] Please note that the links to the images and icons on your page are broken probably due to the changes made🤔.  Kindly rectify and fix this👍